MIL/C02_Basics/S01_Calculating.lean

```diff
@@ -351,12 +351,44 @@ example (a b : ℝ) : (a + b) * (a - b) = a ^ 2 - b ^ 2 := by
 #check mul_sub a b c
 #check add_mul a b c
 #check add_sub a b c
+#check sub_add a b c
+#check sub_self a
 #check sub_sub a b c
 #check add_zero a
 -- QUOTE.
 
 end
 
+-- SOLUTIONS:
```

Comment:
This is my guess at the syntax, so pls don't merge without testing that I didn't bork the page